apps/web/src/utils/getTokenLogoURL.ts

```diff
@@ -7,12 +7,25 @@ const mapping = {
   [ChainId.ETHEREUM]: 'ethereum',
 }
 
+const chainName: { [key: number]: string } = {
+  [ChainId.BSC]: '',
+  [ChainId.ETHEREUM]: 'eth',
+}
+
 const getTokenLogoURL = memoize(
   (token?: Token) => {
     if (token && mapping[token.chainId]) {
-      return `https://assets-cdn.trustwallet.com/blockchains/${mapping[token.chainId]}/assets/${getAddress(
-        token.address,
-      )}/logo.png`
+      const { chainId } = token
```

Comment:
Can we try reuse the function here instead?
https://github.com/pancakeswap/pancake-frontend/blob/develop/packages/uikit/src/components/CurrencyLogo/utils.ts#L39

Reply:
Actually, I had already implemented the same function in `apps/web/src/utils/getTokenLogoURL.ts` initially, but JoJo raised the above suggestion, so I made further modifications to the implementation.